src/deck/guides/konnect.md

```diff
@@ -276,7 +301,28 @@ They are managed entirely by {{site.konnect_short_name}}, so you can't manage th
 
 [Manage application registration](/konnect/dev-portal/applications/enable-app-reg) through the Service Hub to avoid any issues.
 
+{% if_version gte:1.15 %}
+
+### decK keeps targeting {{site.base_gateway}} instead of {{site.konnect_short_name}}
+
+decK can run against {{site.base_gateway}} or {{site.konnect_short_name}}.
+By default, it targets {{site.base_gateway}}, unless a setting tells decK to point to {{site.konnect_short_name}} instead.
+Make sure that one of the {{site.konnect_short_name}} options is present.
```

Comment:
Should there be a list of options here? I wasn't sure what the options were.

Reply:
Ah, that would be referring to the precedence list. I can remove the sentence if it makes more sense without it.